overlord/configstate/configcore/cloud_test.go

```diff
@@ -39,11 +39,6 @@ var _ = Suite(&cloudSuite{})
 
 func (s *cloudSuite) SetUpTest(c *C) {
```

Comment:
If SetUpTest() now just calls the parent SetUpTest() then this entire function can be removed now :)